config.toml

```diff
@@ -28,6 +28,8 @@ weight = 1
     [markup.goldmark.renderer]
       # Enables us to render raw HTML
       unsafe = true
+    [markup.goldmark.parser.attribute]
+      block = true  # used by the summary-table shortcode
```

Comment:
it would be nice to give a quick example of what it actually does. And/or link to the documentation